lib/cassandra_migrations/railtie/tasks.rake

```diff
@@ -65,10 +67,10 @@ namespace :cassandra do
   end
 
   namespace :test do
-    desc 'Load the development schema in to the test keyspace'
+    desc 'Load the development schema in to the test keyspace via a full reset'
     task :prepare do
       Rails.env = 'test'
-      Rake::Task['cassandra:reset'].execute
+      Rake::Task['cassandra:migrate:reset'].execute
```

Comment:
I think this is a very good point, but I worry about backward compatibility. Isn't it possible that someone has a bigger automated process that calls this rake task?